### ISSUE TITLE
Twilio Error 31000: Error creating the offer: Failed to parse SessionDescription.  Failed to parse audio codecs correctly

### ISSUE THREAD

**Issue by dinevillar** [twilio/twilio-client.js#151]
Hello,

I am writing this issue because I have tried debugging my setup for quite a while now but cannot get past this issue.

All of this was working when I was using Twilio Client v1.3 but when I tried upgrading to 1.10. This issue occurs only when I call Twilio.Device.connect. For now I have rolledback to 1.3 and everything works fine.

This is my setup params:
`    'debug': true,
    'closeProtection': true,
    'codecPreferences': ['opus', 'pcmu']`

I have tested on Google Version 80.0.3987.163 (Official Build) (64-bit). Same thing is happening on Firefox 74

This is my webrtc offer
![2020-04-24 02_28_28-](https://user-images.githubusercontent.com/829057/80136129-df598100-85d3-11ea-9310-a90ff7dc090d.png)

This is my console log
![2020-04-24 02_30_17-Settings](https://user-images.githubusercontent.com/829057/80136233-057f2100-85d4-11ea-9932-ed2ac608ef3a.png)

I hope you can help me as I have been trying to fix this for weeks already.

Thank you for this awesome library!

**Comment by charliesantos** [comment]
Hi @dinevillar , thank you for submitting. We've seen this issue before from a different customer and it turned out, they have a library that is changing how the built-in array functions work, which is causing some issues related to how we parse SDP.

Before we dig deeper, can you please verify that it is not related to a 3rd party library by running our quickstart https://github.com/TwilioDevEd/client-quickstart-js?

Also, do you have a public repo or a testapp where I can run and see this issue?

I'm looking forward to your reply!

Thank you,
Charlie

**Comment by dinevillar** [comment]
Hello @charliesantos ,

I found what is conflicting with the twilio library.

I ran the quickstart and manually included my libraries one-by-one then when I included prototype.js (1.7.3) it replicated the same error I am having. Apparently it is overriding `Array.from` on line 1114 `Array.from = $A;`. I am not sure what is the repurcussion of commenting out the line so I just made a reference to Array.from before including prototype.js then resetting it after it has been included. 

Putting this code as a reference for users in the future:
`    <script>
      var wat = Array.from;
    </script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/prototype/1.7.2/prototype.js"></script>
    <script>
      Array.from = wat;
    </script>`

Anyway, thank you very much for pointing me to it.
